TrafficCapture/dockerSolution/src/main/docker/docker-compose.yml

```diff
@@ -97,12 +116,14 @@ services:
       - COMPARISONS_DB_LOCATION=/shared/comparisons.db
     command: /bin/sh -c 'cd trafficComparator && pip3 install --editable ".[data]" && jupyter notebook --ip=0.0.0.0 --port=8888 --no-browser --allow-root'
 
-  opensearch-benchmarks:
-    image: 'migrations/open_search_benchmark:latest'
+  migration_console:
+    image: 'migrations/migration_console:latest'
     networks:
       - migrations
+    volumes:
+      - sharedReplayerOutput:/shared-replayer-output
     depends_on:
-      - captureproxy
+      - capture_proxy_es
```

Comment:
What is the actual dependency here? Is there shared data/env vars/anything like that?

eventually, we very likely want to set up the migration console/orchestrator before the user installs the capture proxy.